doc/content/en/docs/++version++/Specification/_index.md

```diff
@@ -872,6 +877,11 @@ The `local-timestamp-micros` logical type represents a timestamp in a local time
 
 A `local-timestamp-micros` logical type annotates an Avro `long`, where the long stores the number of microseconds, from 1 January 1970 00:00:00.000000.
 
+### Local timestamp (nanosecond precision)
+The `local-timestamp-nanos` logical type represents a timestamp in a local timezone, regardless of what specific time zone is considered local, with a precision of one nanosecond.
+
+A `local-timestamp-nanos` logical type annotates an Avro `long`, where the long stores the number of nanoseconds, from 1 January 1970 00:00:00.000000.
```

Comment:
(not critical) add three zeros to illustrate billionths-of-seconds: `1 January 1970 00:00:00.000000000`

Reply:
That's a good one, thanks for pointing out 👍